controlplane/kubeadm/internal/workload_cluster_conditions_test.go

```diff
@@ -42,15 +43,19 @@ import (
 
 func TestUpdateEtcdConditions(t *testing.T) {
 	tests := []struct {
-		name                             string
-		kcp                              *controlplanev1.KubeadmControlPlane
-		machines                         []*clusterv1.Machine
-		injectClient                     client.Client // This test is injecting a fake client because it is required to create nodes with a controlled Status or to fail with a specific error.
-		injectEtcdClientGenerator        etcdClientFor // This test is injecting a fake etcdClientGenerator because it is required to nodes with a controlled Status or to fail with a specific error.
-		expectedKCPCondition             *clusterv1.Condition
-		expectedKCPV1Beta2Condition      *metav1.Condition
-		expectedMachineConditions        map[string]clusterv1.Conditions
-		expectedMachineV1Beta2Conditions map[string][]metav1.Condition
+		name                                      string
+		kcp                                       *controlplanev1.KubeadmControlPlane
+		machines                                  []*clusterv1.Machine
+		injectClient                              client.Client // This test is injecting a fake client because it is required to create nodes with a controlled Status or to fail with a specific error.
+		injectEtcdClientGenerator                 etcdClientFor // This test is injecting a fake etcdClientGenerator because it is required to nodes with a controlled Status or to fail with a specific error.
+		expectedKCPCondition                      *clusterv1.Condition
+		expectedKCPV1Beta2Condition               *metav1.Condition
+		expectedMachineConditions                 map[string]clusterv1.Conditions
+		expectedMachineV1Beta2Conditions          map[string][]metav1.Condition
+		expectedEtcdMembers                       []*etcd.Member
```

Comment:
Unused, wondering if it would be good to use it / validate it (maybe not the entire members, but maybe the names?)